dart/lib/src/protocol/contexts.dart

```diff
@@ -11,107 +13,170 @@ import 'runtime.dart';
 /// the current HTTP request.
 ///
 /// See also: https://docs.sentry.io/development/sdk-dev/event-payloads/contexts/.
-class Contexts {
-  const Contexts({
-    this.device,
-    this.operatingSystem,
-    this.runtimes,
-    this.app,
-    this.browser,
-    this.gpu,
-  });
+class Contexts extends MapView {
```

Comment:
just out of curiosity, why not `HashMap`? what's the difference between them?

Reply:
Extending HashMap requires implementing most of its members. MapView is only a wrapper around an initial Map, which seemed simpler in this case.

Reply:
I see, thanks.
Should we at least set its type then? `extends MapView<String, dynamic> `

Reply:
✅